### DISCUSSION TITLE
[Task]: Implement RD++ Paper

### DISCUSSION THREAD

**Discussion by willyfh** [open-edge-platform/anomalib#1605] [Feature Requests]
### What is the motivation for this task?

Implementation of RD++, one of the sota anomaly detection models published in CVPR 2023.

### Describe the solution you'd like

📄 [Paper](https://openaccess.thecvf.com/content/CVPR2023/papers/Tien_Revisiting_Reverse_Distillation_for_Anomaly_Detection_CVPR_2023_paper.pdf)
🧑‍💻 [Code](https://github.com/tientrandinh/Revisiting-Reverse-Distillation)

**Comment by samet-akcay** [comment]
Thanks for creating this @willyfh. Yeah this paper is on our radar and we would like to implement it.

**Comment by cjy513203427** [comment]
Hi. I am working on integrating RD++ with anomalib. I have the source code of RD++ deployed on my computer. I tried py36,37,38,310, only py39 worked. Installing through requirements.txt is not friendly. So far I have added loss and projection layer to forked anomalib repo based on source and paper. I am testing with recommended epochs from paper author. Soon I will make a pull request but I still have some questions like how to add noise for MVTec and so on. @willyfh @samet-akcay
